### PR TITLE
Update the empty state illustration

### PR DESCRIPTION
**Pull request by ahamed**

### CODE REVIEW

File: classes/Tutor.php [hunk 1]
```diff
@@ -560,11 +560,6 @@ public function __construct() {
 		 */
 		new Ecommerce();
 
-		/**
-		 * Tutor AI generation controller
-		 */
-		new MagicAI();
```
Comment (sazedul-haque) [inline classes/Tutor.php:566]: @ahamed Vai, this seems irrelevant to this PR. Was it intentional or a accidental?

Comment (ahamed) [reply classes/Tutor.php:566]: Intentional.

**Review by sazedul-haque — APPROVED** [review]
